README.rst

```diff
@@ -151,10 +203,96 @@ display add the new tab for On Task:
+settings of the platform. You can find more information about this in the
+`Open edX documentation`_.
+
+.. _Open edX documentation: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/enable_custom_course_settings.html
+
+
+View from the Learning Management System (LMS)
+**********************************************
+
+When the instructor accesses the OnTask tab, they will have the option to
+create an OnTask Workflow in case there is not one configured. Once the
+**Create Workflow** button is clicked, a workflow will be created in OnTask and
+will be named as the current course ID. e.g. ``course-v1:edunext+ontask+demo``.
+
+.. image:: https://github.com/eduNEXT/platform-plugin-ontask/assets/64033729/ab2c2623-1b74-479e-b8f2-7d37cb4b9a64
+
+The instructor can then view the OnTask workflow related to the course. From
+there the instructor can create, import/export or execute all actions. In the
+upper left corner there will be a button called **Load Data**. Once pressed, an
+asynchronous task will be executed that will generate a data summary of the
+course and load it into a table in the OnTask Workflow. Each time the button is
+pressed, the data will be overwritten. The instructor can then use this data to
+create different actions.
+
+.. image:: https://github.com/eduNEXT/platform-plugin-ontask/assets/64033729/e59f763f-27e6-43aa-bde4-4e80ec41674e
+
+Once the data has been loaded, the instructor can go to **Table** > **View
+data**. There the instructor find the table with all the data summary of the
+course.
+
+.. image:: https://github.com/eduNEXT/platform-plugin-ontask/assets/64033729/6dbb34b7-9829-4bb8-9526-f1b8d5d207cb
+
+
+Create a Custom Data Summary Backend
+*********************************************
+
+By default, the data summary that is loaded into the OnTask table is generated
+from the course completion data. This are the columns that are loaded into
+OnTask:
+
+- **ID** *(Integer)*: An incremental ID.
+- **User ID** *(Integer)*: ID of the user.
+- **Username** *(String)*: Username of the user.
+- **Email** *(String)*: Email of the user.
+- **Course ID** *(String)*: ID of the course.
+- **Unit ID** *(String)*: ID of the unit.
+- **Unit Name** *(String)*: Name of the unit.
+- **Is Completed** *(Boolean)*: If the unit is completed.
+
+You can create a custom data summary backend to customize the data summary
+that is loaded into the OnTask table. To do this, follow these steps:
+
+1. Create a new file in the ``datasummary`` directory with the name of the
+   backend, e.g., ``custom.py``
+2. Create a class that inherits from ``DataSummary``, e.g.,
+   ``CustomDataSummary(DataSummary)``
+3. Implement the ``get_data_summary`` method to return the data summary. The
+   method must return a dictionary where each key is the column name and the
+   value is other dictionary with the id as key and the value as the value of
+   the column. You can check an example in the ``dummy.py`` file.
```

Comment:
Great! Can we add an example of how this would look like?

Reply:
Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/dc607bd67737935a5987f02caefa61875f4dfb6b